content/docs/static-type-checking.md

```diff
@@ -259,40 +258,42 @@ We'll address this in two steps:
 
-### Running TypeScript {#running-typescript}
+Si vous avez bien suivi les instructions ci-dessus, vous devriez pouvoir exécuter TypeScript pour la première fois.
 
-If you followed the instructions above, you should be able to run TypeScript for the first time.
+Si vous utilisez Yarn, exécutez :
 
 ```bash
 yarn build
 ```
 
-If you use npm, run:
+Si vous utilisez npm, exécutez :
 
 ```bash
 npm run build
 ```
 
-If you see no output, it means that it completed successfully.
+Si aucun message ne s'affiche, ça veut dire que tout a bien fonctionné.
 
 
-### Type Definitions {#type-definitions}
-To be able to show errors and hints from other packages, the compiler relies on declaration files. A declaration file provides all the type information about a library. This enables us to use javascript libraries like those on npm in our project. 
+### Définitions de types {#type-definitions}
+Afin de pouvoir montrer les erreurs et indications des autres modules, le compilateur a besoin de fichiers de déclaration. Un fichier de déclaration contient toutes les informations sur les types d'une bibliothèque. Ça nous permet d'utiliser des bibliothèques javascript telles que celles que l'on trouve sur npm dans notre projet.
 
-There are two main ways to get declarations for a library:
+Les deux principales façons d'obtenir les déclarations de types d'une bibliothèque sont les suivantes :
 
-__Bundled__ - The library bundles its own declaration file. This is great for us, since all we need to do is install the library, and we can use it right away. To check if a library has bundled types, look for an `index.d.ts` file in the project. Some libraries will have it specified in their `package.json` under the `typings` or `types` field.
+__Inclue__ - La bibliothèque comprend déjà un fichier de déclaration. Ça nous arrange, il nous suffira d'installer la bibliothèque et nous pourrons l'utiliser sans tarder. Pour vérifier qu'une bibliothèque contient les types, cherchez un fichier `index.d.ts` dans le projet. Certaines bibliothèques l'annoncent dans leur `package.json` sous le champ `typings` ou `types`.
 
-__[DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped)__ - DefinitelyTyped is a huge repository of declarations for libraries that don't bundle a declaration file. The declarations are crowd-sourced and managed by Microsoft and open source contributors. React for example doesn't bundle its own declaration file. Instead we can get it from DefinitelyTyped. To do so enter this command in your terminal.
+__[DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped)__ - DefinitelyTyped est un énorme dépôt de déclarations pour les bibliothèques qui n'ont pas de fichier de déclaration. Les déclarations sont faites par la foule et gérées par Microsoft et les contributeurs open source. React par exemple n'a pas de fichier de déclaration. On peut l'obtenir sur DefinitelyTyped. Pour y arriver, exécutez cette commande dans votre terminal.
```

Comment:
```suggestion
__[DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped)__ - DefinitelyTyped est un énorme dépôt de déclarations pour les bibliothèques qui n'ont pas de fichier de déclarations. Les déclarations sont faites par la tout le mode et gérées par Microsoft et les contributeurs open source. React par exemple n'a pas de fichier de déclarations. On peut l'obtenir sur DefinitelyTyped. Pour y arriver, exécutez cette commande dans votre terminal.
```

Reply:
j'ai changé foule par tout le monde
je ne suis pas un grand fan de ma version non plus, mais foule, surtout dans l'IT, n'est pas un mot que j ai vu utilisé a voir avec d'autres reviewer aussi

Reply:
J'ai hésité sur ce terme. J'avais pensé à "de façon participative", qu'en penses-tu ?